find_similar/tokenize.py

```diff
@@ -29,14 +29,7 @@
 def add_nltk_stopwords(language: str, stop_words=None):
     if stop_words is None:
         stop_words = STOP_WORDS_NO_LANGUAGE
-    try:
-        stopwords_with_language = stopwords.words(language)
-    except LookupError:
-        nltk.download('stopwords')
-        nltk.download('punkt')
-        stopwords_with_language = stopwords.words(language)
-    except OSError:
-        raise LanguageNotFound(language)
+    stopwords_with_language = get_stopwords_from_nltk(language)
```

Comment:
This function define at the bottom. May be will be better to define it above for better code reading.